test/data/navigator/useragentdata-brand.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<!-- navigator.userAgentData brand test -->
+<html>
+  <head>
+    <title>fail</title>
+    <meta charset="utf-8">
+</head>
+<body>
+  <script>
+    navigator.userAgentData.getHighEntropyValues(
+      ["fullVersionList"]).then(ua => {
+        if (ua.fullVersionList.length == 3) {
```

Comment:
I think we should verify that it contains all three so we know we didn't accidentally mess up the original values

Reply:
added in https://github.com/brave/brave-core/commit/d186816a8ec15adde1d723ae3bdf1664db604041